MAINTAINERS.md

```diff
@@ -135,16 +120,15 @@ During your review, consider the following points:
 
 ## Workflow for merging Pull Requests
 
-Check which branch the PR was made against — patches made against the `develop` branch should only
-be merged to `develop`. Patches made against the `master` branch may be merged to only `develop`, or
-both `develop` AND `master`. The criteria to use in this latter situation is: does the patch
-introduce anything new to the API? does it potentially mark a change in behavior of any sort? If the
-answer to either of these is "yes", merge only to the `develop` branch.
+Pull Requests should only be merged into the branches they were made against. When a PR contains a 
+feature and was made against an already released branch, you should work with the contributor to target
+it against the next release branch. The PR should have a Milestone matching the next release of the
+target branch (for example milestone `1.0.1` for branch `1.0.x`).
```

Comment:
This workflow works pretty much the opposite now.

Since the default branch is the latest release branch, the more common scenario is that somebody will submit a bugfix against the latest release branch, when it should go against the previous. In those cases, you'll need to re-target the PR and help the user rebase or cherry-pick against the previous release branch.